server/src/test/java/org/elasticsearch/node/NodeTests.java

```diff
@@ -336,7 +334,7 @@ public void testCloseOnLeakedStoreReference() throws Exception {
 
     public void testStartOnClosedTransport() throws IOException {
         try (Node node = new MockNode(baseSettings().build(), basePlugins())) {
-            node.injector().getInstance(HttpServerTransport.class).close();
+            node.prepareForClose();
```

Comment:
Nit: it is a tiny bit different, as `prepareForClose()` invokes `HttpServerTransport#stop()`, which does "less". If the test passes I think it's OK, but possibly needs renamig to `testStartOnStoppedTransport`?

Reply:
This PR modifies `prepareForClose` to call `close` instead

Reply:
Yes, just figured out: I was looking at the wrong revision. Looks good!